client/dashboard/utils/site-url.ts

```diff
@@ -47,3 +48,14 @@ export function getSiteEditUrl( site: Site, isSiteUsingBlockTheme?: boolean ) {
 
 	return addQueryArgs( `${ siteAdminUrl }customize.php`, queryArgs );
 }
+
+/**
+ * Returns the URL for the site visibility settings page.
+ */
+export function getVisibilityURL( site: Site ) {
```

Comment:
Nit: maybe update the URL to `getSiteVisibilitySettingsUrl()` to be consistent with the rest of the functions in this file 😄